core/lib/mno_enterprise/platform_adapters/nex_adapter.rb

```diff
@@ -1,19 +1,82 @@
 # frozen_string_literal: true
-
-# gem 'nex_client'
-# require 'nex_client'
-# SELF_NEX_API_KEY="some-api-key"
-# SELF_NEX_API_ENDPOINT="https://api-nex-uat.maestrano.io"
+gem 'nex_client', '~> 0.16.0'
```

Comment:
No. It's up to the host app to add it in its Gemfile.
I'll add a check when selecting the adapter